src/components/sprint-view-table/tasks-table.tsx

```diff
@@ -25,87 +36,64 @@ interface Props {
 
-  /**
-   * Gather tasks and time entries when project is available and update reload state
-   */
-  useEffect(() => {
-    if (project && open) {
-      getTasksAndTimeEntries();
-    }
-  }, [project, open, filter]);
+  const rows = phase.map((phase) => {
+    const actualWorkHours = getWorkHour(
+      phase.severaPhaseId || "Severa Phase Id not found",
+      workHours
+    );
+    return {
+      id: phase.severaPhaseId,
+      title: phase.name,
+      estimateWorkHours: phase.workHoursEstimate,
+      startDate: phase.startDate?.toISOString().split("T")[0],
+      deadLine: formatDateSevera(phase.deadline?.toISOString() || ""),
+      actualWorkHours: actualWorkHours || "0",
+    };
+  });
 
   /**
-   * Handle loggenInPersonId change
+   * Get Tasks and WorkHours for tasks
    */
-  useEffect(() => {
-    setTasks([]);
-    setOpen(false);
-  }, [loggedInPersonId]);
-
-  /**
-   * Get tasks and total time entries
-   */
-  const getTasksAndTimeEntries = async () => {
+  const getTasksAndWorkHours = async () => {
     setLoading(true);
-    if (!tasks.length || !timeEntries.length) {
+    if (!phase?.length) {
       try {
-        const fetchedTasks = !loggedInPersonId
-          ? await tasksApi.listProjectTasks({ projectId: project.id })
-          : await tasksApi
-              .listProjectTasks({ projectId: project.id })
-              .then((result) =>
-                result.filter((task) => task.assignedPersons?.includes(loggedInPersonId || 0))
-              );
-
-        setTasks(fetchedTasks);
-        const fetchedTimeEntries = await Promise.all(
-          fetchedTasks.map(async (task) => {
-            try {
-              if (project.id) {
-                const totalTimeEntries = await timeEntriesApi.listProjectTimeEntries({
-                  projectId: project.id,
-                  taskId: task.id
-                });
-                let totalMinutes = 0;
-                totalTimeEntries.forEach((timeEntry: TimeEntries) => {
-                  if (loggedInPersonId && timeEntry.person === loggedInPersonId) {
-                    totalMinutes += timeEntry.timeRegistered || 0;
-                  }
-                  if (!loggedInPersonId) {
-                    totalMinutes += timeEntry.timeRegistered || 0;
-                  }
-                });
-                return totalMinutes;
-              }
-            } catch (error) {
-              const message: string = strings
-                .formatString(
-                  strings.sprintRequestError.fetchTasksError,
-                  task.id || `${strings.sprintRequestError.fetchTaskIdError}`,
-                  error as string
-                )
-                .toString();
-              setError(message);
-            }
-            return 0;
-          })
-        );
-        setTimeEntries(fetchedTimeEntries);
+        const fetchedTasks = await phaseApi.getPhasesBySeveraProjectId({
+          severaProjectId: project.project?.severaProjectId || "",
```

Comment:
It's odd to have project.project again perhaps the project in Props shoudl be called ResourceAllocations to reflect its actual type?